R/logregbin.b.R

```diff
@@ -288,8 +288,10 @@ logRegBinClass <- if (requireNamespace('jmvcore')) R6::R6Class(
                 r2mf <- 1 - dev/nullDev
                 r2cs <- 1 - exp(-(nullDev - dev) / n)
                 r2n <- r2cs / (1 - exp(-nullDev / n))
+                r2t <- unname(diff(tapply(predict(self$models[[i]], type="response"),
+                                          self$models[[i]]$y, mean, na.rm=TRUE)))
```

Comment:
Maybe this would be a little bit more descriptive if it was calculated in two steps (with some added formatting):
```suggestion
                meanFittedProbs <- tapply(
                    self$models[[i]]$fitted.values, self$models[[i]]$y, mean, na.rm=TRUE
                )
                r2t <- unname(diff(meanFittedProbs))
```

Reply:
You're right, two steps make it more readable

Reply:
I still see this as a oneliner in the code at the moment. Did you decide this was not a good idea anyway or did something go wrong with the commits?